doc/extdev/index.rst

```diff
@@ -147,7 +147,7 @@ the individual nodes of each doctree and produces some output in the process.
    that checks external links does not need anything more than the parsed
    doctrees and therefore does not have phases 2--4.
 
-To see an example of application, refer to :doc:`../development/tutorials/todo`.
+To see an example of application, refer to :ref:`tutorial-extend-build`.
```

Comment:
Is the migration from `:doc:` to `:ref:` for hyperlinking to avoid the need to keep track of file paths within the project?

Reply:
yes, I don't feel you should ever really be using `doc`, `ref` is a lot more robust for changes in the documentation structure